starters/next-wordpress-starter/pages/examples/pagination/[[...page]].jsx

```diff
@@ -0,0 +1,64 @@
+import { NextSeo } from 'next-seo';
+import { setEdgeHeader } from '@pantheon-systems/wordpress-kit';
+
+import Layout from '../../../components/layout';
+import { Paginator } from '@pantheon-systems/nextjs-kit';
+
+import { getFooterMenu } from '../../../lib/Menus';
+import { paginationPostsQuery } from '../../../lib/PostsPagination';
+
+export default function PaginationExampleTemplate({ menuItems, posts }) {
+	const RenderCurrentItems = ({ currentItems }) => {
+		return currentItems.map((item) => {
+			return (
+				<article
+					key={item.title}
+					className="flex flex-col p-3 w-fit mx-auto mb-10"
+				>
+					<h2 className="justify-start my-auto text-2xl mb-2">{item.title}</h2>
+					<p
+						className="max-w-prose my-2"
+						dangerouslySetInnerHTML={{ __html: item.excerpt }}
+					/>
+				</article>
+			);
+		});
+	};
+	return (
+		<Layout footerMenu={menuItems}>
+			<NextSeo>
+				<title>Pagination example</title>
+				<meta name="description" content="Powered by Pantheon Decoupled Kit" />
+				<link rel="icon" href="/favicon.ico" />
+			</NextSeo>
+			<div className="prose container min-w-full max-w-screen mx-auto">
+				<main>
```

Comment:
I was trying to point out in the last comment that we don't want a `main` tag here because the `<Layout />` already gives us one. I am realizing now that the `next-drupal-starter` has the same issue. We should resolve that in a separate PR. If it's quick you can do it after you update the markup here.

Reply:
@mitchellmarkoff I think this is all we need surrounding the paginator:
```
<div className="prose max-w-screen mx-auto">
					<section className="flex flex-col">
						<h1 className="my-10">Pagination example</h1>
						<Paginator
                                                     ...
						/>
					</section>
			</div>
```